src/plugins/intel_cpu/src/transformations/snippets/x64/pass/set_brgemm_cpu_blocking_params.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace pass {
+
+/**
+ * @interface SetBrgemmCPUBlockingParams
+ * @brief The pass selects optimal M, K and N blocking parameters for BrgemmCPU and sets them to the node.
+ * @ingroup snippets
+ */
+class SetBrgemmCPUBlockingParams: public ov::pass::MatcherPass {
```

Comment:
Not sure if this is the best name for the transformation. Maybe `SelectBrgemmCPUBlockingParams` is better?

Reply:
We have already `SetSoftmaxPorts` 🙂 So seems like it's ugly name but with the sense